### PR TITLE
single camera case

### PR DESCRIPTION
**Pull request by alexlib**
This is a pull request for a single camera case. The main additions are: 

`epi_mm_2D` function is used through Cython bindings
`point_positions` is now a generalized function that points toward a `single_cam_point_positions` (a new function) and the previous `multi_cam_point_positions`
`correspondences` have now a shortcut for a single camera case that returns the values immediately, otherwise continues to standard 3d correspondences

### CODE REVIEW

File: liboptv/src/correspondences.c [hunk 1]
```diff
@@ -4,14 +4,18 @@ Routine:	       	correspondences.c
 
 Author/Copyright:      	Hans-Gerd Maas
 
-Address:	      	Institute of correcteddesy and Photogrammetry
+Address:	      	Institute of Geodesy and Photogrammetry
```
Comment (yosefm) [inline liboptv/src/correspondences.c]: :-)

File: py_bind/optv/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yosefm) [inline py_bind/optv/.vscode/settings.json]: It's in gitignore and yet it's here.

File: py_bind/optv/correspondences.pyx [hunk 3]
```diff
@@ -171,49 +176,116 @@ def correspondences(list img_pts, list flat_coords, list cals,
...
-    # The biz:
-    corresp_buf = corresp(&frm, corrected, 
-        vparam._volume_par, cparam._control_par, calib, match_counts)
-    
     # Distribute data to return structures:
     sorted_pos = [None]*(num_cams - 1)
     sorted_corresp = [None]*(num_cams - 1)
     last_count = 0
     
+    # The biz:
```
Comment (yosefm) [inline py_bind/optv/correspondences.pyx]: I don't know why you switched places, but these variables are only needed after the call to corresp().

File: py_bind/test/test_corresp.py [hunk 4]
```diff
@@ -38,11 +38,11 @@ def test_instantiate(self):
 class TestCorresp(unittest.TestCase):
     def test_full_corresp(self):
         """Full scene correspondences"""
-        print "about to dump core"
+        print("about to dump core")
```
Comment (yosefm) [inline py_bind/test/test_corresp.py]: leftover debug message?

File: py_bind/optv/orientation.pxd [hunk 5]
```diff
@@ -31,3 +35,7 @@ cdef extern from "optv/orientation.h":
         int db_length, double db_weight)
 
 cdef calibration** cal_list2arr(list cals)
+
+cdef extern from "optv/epi.h":
```
Comment (yosefm) [inline py_bind/optv/orientation.pxd]: consider putting it in epipolar.pxd and cimporting from it.

Comment (alexlib) [reply py_bind/optv/orientation.pxd]: done

**Comment by alexlib** [comment]
I mistakenly merged it and now I don't know how to revert the situation. @yosefm ?

**Pull request merged**